src/rely/reporters/JunitReporter.re

```diff
@@ -0,0 +1,62 @@
+let testResultToTestCase = (testResult: Reporter.testResult) => {
```

Comment:
Can we get the copyright header in this file? You can copy it from any other .re file